docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_types.go

```diff
@@ -81,14 +81,17 @@ type CronJobSpec struct {
 	// - "Forbid": forbids concurrent runs, skipping next run if previous run hasn't finished yet;
 	// - "Replace": cancels currently running job and replaces it with a new one
 	// +optional
-	ConcurrencyPolicy ConcurrencyPolicy `json:"concurrencyPolicy,omitempty"`
+	// +kubebuilder:validation:Enum=Allow;Forbid;Replace
```

Comment:
Nit: Maybe better to put this marker on the type, including the enum values docs? In case the type is reused in the API, I prefer not to repeat myself. The default value for the field can stay here, I think.

Reply:
You are right

```
                allOf:
                - enum:
                  - Allow
                  - Forbid
                  - Replace
                - enum:
                  - Allow
                  - Forbid
                  - Replace
                default: Allow
```

I would say that would be nice have implicity but in this case, is odd